src/dependency-builder/build-tree.js

```diff
@@ -228,6 +188,24 @@ function resolveModulePath(nmPath, workingDir, root) {
   return resolveModulePath(nmPath, parentWorkingDir, root);
 }
 
+/**
+ * Resolve package dependencies from package.json
+ *
+ * @param {string} cwd
+ * @param {string []} missing
+ * @param {string} consumerPath
+ * @returns new object with found and missing
+ */
+function resolvePackageDependeces (cwd: string, missing: string[] ) {
```

Comment:
i would change it to findPackagesInPackageJson, this will require few changes:
1. change the first arg to be the packageJson itself
2. change the second arg to be packagesList